app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.players.player import Player
+from typing import List
+
+
+def calculate_team_total_rating(team: List[Player]) -> int:
```

Comment:
you may use built-in `list` object for type annotations as of python 3.9 (it is not a mistake what you did - you can just avoid having an extra import if you do so)